### PR TITLE
Add StandardJS (closes #323)

### LINKED ISSUE #323: Use StandardJS
I think that we could start using [StandardJS](https://standardjs.com/) to have a universal, but [_modern_](http://blog.izs.me/post/2353458699/an-open-letter-to-javascript-leaders-regarding), code style.

### PR DESCRIPTION
**Pull request by j-a-m-l**
This PR adds StandardJS (#323) with the configuration for this project.

I've adapted the code under the `test` folder to that code style too. The rest of the code should be done incrementally because it is composed of a lot of different parts.

### CODE REVIEW

File: test/e2e/scenarios/select-network.js [hunk 1]
```diff
@@ -2,41 +2,39 @@
...
   })
 
-  before(function() {
-    return this.app.client.addCommand('selectNetwork', function(network) {
+  before(function () {
+    return this.app.client.addCommand('selectNetwork', function (network) {
       return this.click('[aria-label="Switch Network"] button')
         .pause(500)
-        .click("//button[contains(text(), '"+network+"')]")
+        .click("//button[contains(text(), '" + network + "')]")
```
Comment (spresnal) [inline test/e2e/scenarios/select-network.js]: Is there an option in standardJS to enforce template literals?

**Comment by j-a-m-l** [comment]
@spresnal In that case I kept it that way because I'm waiting to having more parts refactored (https://github.com/ArkEcosystem/ark-desktop/issues/298) before changing to a more recent Node.js version (https://github.com/ArkEcosystem/ark-desktop/issues/299)

File: test/components/addressbook/addressbook.controller.js [hunk 2]
```diff
@@ -1,24 +1,22 @@
...
   beforeEach(module('arkclient.components'))
 
-  beforeEach( ()=> {
-    inject( (_$compile_, _$rootScope_)=> {
-      $compile = _$compile_
-      $rootScope = _$rootScope_
-      $scope = _$rootScope_.$new()
-    })
-  })
+  // beforeEach( () => {
```
Comment (fix) [inline test/components/addressbook/addressbook.controller.js]: should be removed not commented out

**Comment by luciorubeens** [comment]
Merged manually! +5 👍🏻

**Pull request merged**